bin/propolis-standalone/src/main.rs

```diff
@@ -952,6 +952,42 @@ fn generate_smbios(params: SmbiosParams) -> anyhow::Result<smbios::TableBytes> {
     Ok(smb_tables.commit())
 }
 
+fn generate_bootorder(config: &config::Config) -> anyhow::Result<fwcfg::Entry> {
+    let names = config.main.boot_order.as_ref().unwrap();
+
+    let mut order = fwcfg::formats::BootOrder::new();
+    for name in names.iter() {
+        let dev = config
+            .devices
+            .get(name)
+            .ok_or(anyhow::anyhow!("Could not find device: {name}"))?;
```

Comment:
nitpicky: this error message doesn't really indicate that they occurred while trying to generate the boot order. maybe we should say something like 
```suggestion
            .ok_or(anyhow::anyhow!("Could not find boot device: {name}"))?;
```
or, change the caller to add `.context("Failed to generate boot order from config file")` or something to the call to `generate_bootorder`, so that the user sees an error indicating which part of the config file was invalid, before indicating _why_ it was invalid.